test/ext/test_DataInterpolations.jl

```diff
@@ -0,0 +1,71 @@
+
+using SparseConnectivityTracer
+using DataInterpolations
+using DataInterpolations: AbstractInterpolation
+using Test
+
+# Categorize Interpolations by type of differentiability
+interpolations_z = (ConstantInterpolation,)
+interpolations_f = (LinearInterpolation,)
+interpolations_s = (
+    QuadraticInterpolation,
+    LagrangeInterpolation,
+    AkimaInterpolation,
+    QuadraticSpline,
+    CubicSpline,
+)
+
+u = [1.0, 2.0, 5.0]
+t = [0.0, 1.0, 3.0]
+
+@testset "Jacobian" begin
+    @testset "$M" for M in (TracerSparsityDetector, TracerLocalSparsityDetector)
+        method = M()
+        J(f, x) = jacobian_sparsity(f, x, method)
+
+        @testset "Non-differentiable" begin
```

Comment:
Might be nitpicking, but what do you mean here by non-differentiable? Sure, the interpolation is not differentiable in the data points, but I think most people expect the derivative to be 0 everywhere.